crates/ide-completion/src/completions/attribute.rs

```diff
@@ -86,10 +86,26 @@ pub(crate) fn complete_attribute_path(
     acc: &mut Completions,
     ctx: &CompletionContext<'_>,
     path_ctx @ PathCompletionCtx { qualified, .. }: &PathCompletionCtx,
-    &AttrCtx { kind, annotated_item_kind }: &AttrCtx,
+    &AttrCtx { kind, annotated_item_kind, ref derive_helpers }: &AttrCtx,
 ) {
     let is_inner = kind == AttrKind::Inner;
 
+    for (derive_helper, derive_name) in derive_helpers {
+        let mut item = CompletionItem::new(
+            SymbolKind::Attribute,
+            ctx.source_range(),
+            format!("{derive_helper}(…)"),
+            ctx.edition,
+        );
+        if let Some(cap) = ctx.config.snippet_cap {
+            item.insert_snippet(cap, format!("{derive_helper}($0)"));
+        } else {
+            item.insert_text(format!("{derive_helper}("));
+        }
```

Comment:
Derive helpers don't need to accept a token tree, the can be bare `#[helper]` or kv style `#[helper = 0]`, so I don't think we should complete these with `()`

Reply:
Oh I thought they cannot. I will fix that in a minute.

I really wish proc-macro authors had some way to let r-a analyze their grammar...